### DISCUSSION TITLE
any diffrence between reel and video post

### DISCUSSION THREAD

**Discussion by Ashish-Coder12** [subzeroid/aiograpi-rest#43] [Q&A]
is there any way to differentiate between reel and video post in user media section not by individual post

**** not  by  individual post  *****

**Comment by adw0rd** [comment]
<img width="555" alt="image" src="https://user-images.githubusercontent.com/546889/179003431-670e837d-6e2c-4a81-a5db-5351f21a2d1e.png">

https://adw0rd.github.io/instagrapi/usage-guide/media.html
